autode/opt/coordinates/base.py

```diff
@@ -232,7 +232,7 @@ def update_h_from_old_h(
         for update_type in hessian_update_types:
             updater = update_type(
                 h=old_coords._h,
-                s=self.raw - old_coords.raw,
+                s=np.array(self) - np.array(old_coords),
```

Comment:
why?

Reply:
The hessian update basically only acts on the actual coordinates and gradients and not the lagrangian components

Reply:
ah gotcha. I missed the overridden `raw` property 👌🏼